apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -5299,24 +5324,13 @@ sc_ws_timeout_open_(Config) ->
 sc_ws_attach_initiator(Config) ->
     #{initiator := #{pub_key  := Pubkey,
                      priv_key := Privkey}} = proplists:get_value(participants, Config),
-    attach({Pubkey, Privkey}, "simple_auth", "authorize", ["42"]),
+    attach({Pubkey, Privkey}, "simple_auth", "authorize", [?SIMPLE_AUTH_GA_SECRET]),
```

Comment:
Could you please consider PR #2510 - it aims at polishing this area of the test.